include/msvc/wx/setup.h

```diff
@@ -65,6 +65,10 @@
         #define wxCOMPILER_PREFIX vc120
     #elif _MSC_VER == 1900
         #define wxCOMPILER_PREFIX vc140
+    #elif _MSC_VER >= 1910 && _MSC_VER < 1920
+        #define wxCOMPILER_PREFIX vc141
+    #elif _MSC_VER >= 1920 && _MSC_VER < 1930
+        #define wxCOMPILER_PREFIX vc141
```

Comment:
This should be `vc142` unless I'm missing something?

Reply:
Yes it should and it was. The only thing I can think of is I had two copies opened (notepad in windows) and saved the above copy when I closed everything down. I just rechecked an the widgets demo I built requires the 142 dll's.

How do I fix this, just push a new commit?

Reply:
I think you should just push the (correct) commit directly to 3.0 branch, this is not tested by the CI builds anyhow, and close this PR.

Reply:
Bare with me as I haven't done this before. Is the procedure:
1. Checkout WX_3_0_BRANCH
2. Make and commit the change
3. Push to wxWidgets WX_3_0_BRANCH

Reply:
You should be able to just merge (it should "fast forward", i.e. not really create any real merges) your existing `wx305_Prep` into 3.0 by doing
```
git checkout -b 3.0 origin/WX_3_0_BRANCH
git merge --ff-only wx305_Prep
git push
```
(if you already have your own local equivalent of "3.0" branch, please just use it instead, no need to create a new one).

But if you feel more comfortable with me doing this, you can also push to this branch and I'll merge it myself.

Reply:
I will push this branch and let you to it.

Reply:
It's been a while and I think I have missed an intermediate step?. I have now a commit to my first commit. Let me know what to do to make life simpler for you.

Reply:
I've squashed the commits myself, no problem.